notebooks/official/tensorboard/tensorboard_custom_training_with_prebuilt_container.ipynb

```diff
@@ -0,0 +1,968 @@
+{
```

Comment:
<p>The Cloud Storage bucket is used to store the following:</p><p> </p><ul><li>Your training code distribution (details below).</li><li>The outputs your training code generates, including TensorBoard logs.</li></ul><p> </p><p>The bucket must be regional and not multi-regional or dual-regional, and the following resources must also be in same region: </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/603/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>